### PR TITLE
First pass at migrating the code to use tiled

### PR DESCRIPTION
**Pull request by cryos**
This uses tiled, pretty much replicates all core with a hack to show the
last twenty results. Still no angles....

### CODE REVIEW

**Comment by cjh1** [comment]
Thanks @cryos I will give this a try tomorrow.

**Comment by cryos** [comment]
Working with Dan on getting the angles now, this works but is has a ways to go.

File: tomviz/python/tomviz/io/_databroker.py [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 from vtk import vtkImageData
 
 _installed = False
+DEFAULT_URL = "https://tiled.nsls2.bnl.gov"
+TILED_URL = os.getenv("TILED_URL", DEFAULT_URL)
 try:
-    from databroker import catalog
+    from tiled.client import from_uri
+    c = from_uri(TILED_URL, "dask")
     from databroker.queries import TimeRange
```
Comment (cjh1) [inline tomviz/python/tomviz/io/_databroker.py:14]: Can we get rid of this databroker import?

**Comment by cjh1** [comment]
@cryos This is ready for you to take a look at.

File: tomviz/python/tomviz/io/_databroker.py [hunk 2]
```diff
@@ -1,12 +1,15 @@
+import os
 import numpy as np
-
 import tomviz.utils
 
 from vtk import vtkImageData
 
 _installed = False
+DEFAULT_URL = "https://tiled.nsls2.bnl.gov"
+TILED_URL = os.getenv("TILED_URL", DEFAULT_URL)
```
Comment (cryos) [inline tomviz/python/tomviz/io/_databroker.py:9]: I wonder if we might make beamline settable with env variables like this, but in the C++ is certainly more than enough to work on this tomorrow as we also tie to the andor image

**Review by cryos — COMMENTED** [review]
I think this looks good, I haven't had a chance to build and test myself. I can do that this afternoon, but given the time merging now, building, and letting me test in the full Conda environment later is likely best. Thanks for jumping on this Chris!

File: tomviz/python/tomviz/io/_databroker.py (hunk 2 above)
Comment (cjh1) [reply tomviz/python/tomviz/io/_databroker.py:9]: Sure, I can do that

**Pull request merged**